pkgs/servers/portunus/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "portunus";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "majewsky";
+    repo = "portunus";
+    # rev = "v${version}";
+    rev = "99dc0e3d5a84fac988759df8ba2b8cd54ffdc0fb";
+    sha256 = "sha256-CLjpiebQdEWuo5ae5KcBrzLs1hzZHiBIWOpaXdPIJAc=";
+  };
+
+  vendorSha256 = null;
+
+  postInstall = ''
+    mv $out/bin/{,portunus-}orchestrator
+    mv $out/bin/{,portunus-}server
+  '';
+
+  meta = with lib; {
+    description = "Self-contained user/group management and authentication service";
+    homepage = "https://github.com/majewsky/portunus";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ majewsky SuperSandro2000 ];
```

Comment:
@majewsky do you want to maintain this package or should I alone?

Reply:
You can leave me in. I will probably make PRs for version upgrades when I release a new version upstream.

Reply:
Then I need an entry in the maintainers list from you https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L12291-L12297

Reply:
```nix
{
  email = "majewsky@gmx.net";
  github = "majewsky";
  githubId = 24696;
  name = "Stefan Majewsky";
}
```